pkgs/development/libraries/libbfio/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, fetchurl
+, fetchpatch
+, stdenv
+, gettext
+, libtool
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  version = "20221025";
+  pname = "libbfio";
+
+  src = fetchurl {
+    url = "https://github.com/libyal/libbfio/releases/download/${version}/libbfio-alpha-${version}.tar.gz";
+    hash = "sha256-GBS9oVZxdvo4Sv8DaZU0t97aN1oHktkvAsPpemLlOyc=";
+  };
```

nit: consider using `finalAttrs` and `fetchzip`

hash might need to be recalculated.

I also flipped the order of `pname` and `version`, but I don't consider it important.

```suggestion
, fetchzip
, fetchpatch
, stdenv
, gettext
, libtool
, pkg-config
}:

stdenv.mkDerivation (finalAttrs: {
  pname = "libbfio";
  version = "20221025";

  src = fetchzip {
    url = "https://github.com/libyal/libbfio/releases/download/${finalAttrs.version}/libbfio-alpha-${finalAttrs.version}.tar.gz";
    hash = "sha256-GBS9oVZxdvo4Sv8DaZU0t97aN1oHktkvAsPpemLlOyc=";
  };
```

---

@h7x4 fixed and changed hash correctly.